src/Goerli-DssSpell.t.sol

```diff
@@ -163,7 +164,7 @@ contract DssSpellTest is GoerliDssSpellTestBase {
     }
 
     function test_use_eta() public {
-        hevm.warp(1606161600); // Nov 23, 20 UTC (could be cast Nov 26)
+        hevm.warp(1638302400); // Nov 30, 20 UTC (could be cast Dec 2)
```

Comment:
Incorrect date in comment, but I don't think the test needed to be changed.

Reply:
It would be Dec 3 right?

Either way, I reverted this change locally and ran the tests again. They are passing without issues.

TBH I don't quite understand yet what this specific test case is doing.